pkg/v1/tarball/write.go

```diff
@@ -88,6 +88,57 @@ func MultiRefWrite(refToImage map[name.Reference]v1.Image, w io.Writer) error {
 	tf := tar.NewWriter(w)
 	defer tf.Close()
 
+	m, err := processImages(refToImage, tf)
```

Comment:
This diff ended up being really drastic :/ 

This is super pedantic, but if it's possible to rearrange the ordering of some of these functions to help git minimize the diff, it will make me a lot happier in the future when I `git blame` this file 😄

Reply:
No problem.

Reply:
That should do it.